README.md

```diff
@@ -325,10 +331,11 @@ $stk = new STK($engine);
 $response = $stk->request(10)
     ->from(254722000000)
     ->usingReference('some reference', 'Test Payment')
+    ->setCommand(CUSTOMER_PAYBILL_ONLINE)
     ->push();
 
 /****** OR ********/
-$response = $stk->push(10, 254722000000, 'some reference', 'Test Payment');
+$response = $stk->push(10, 254722000000, 'some reference', CUSTOMER_PAYBILL_ONLINE, 'Test Payment');
```

Comment:
move the `CUSTOMER_PAYBILL_ONLINE` to be the last argument and also state that it's optional, giving a default of  `CustomerPayBillOnline`